pages/common/vgmstream_cli.md

```diff
@@ -0,0 +1,29 @@
+# vgmstream CLI decoder
+
+> Play a wide variety of audio formats used in video games.
+> Converts playable files to wav.
+> More information: <https://vgmstream.org/doc/USAGE>.
+
+- Decode a adc file to wav. (Default output name is `input.wav`):
+
+`vgmstream_cli {{path/to/input.adc}} -o {{path/to/output.wav}}`
+
+- Print metadata without decoding the audio:
+
+`vgmstream_cli {{path/to/input.adc}} -m`
+
+- Decode audio without loops:
+
+`vgmstream_cli {{path/to/input.adc}} -o {{path/to/output.wav}} -i`
+
+- Decode with three loops, then add a 3s delay followed by a 5s fadeout:
+
+`vgmstream_cli {{path/to/input.adc}} -o {{path/to/output.wav}} -l 3.0 -f 5.0 -d 3.0`
+
+- Convert multiple files to `bgm_(name).wav` (Default is `?f.wav`):
```

Comment:
I don´t understand what does this default filename.

Reply:
The default filename is in case the `-o` is omited.
I included the `?f` in the example `-o` value below, to illustrate how it works as a wildcard. 

```suggestion
- Convert multiple files to `bgm_(original name).wav` (Default `-o` pattern is `?f.wav`):
```

Reply:
I think the default output file is already clear from the previous examples.
```suggestion
- Convert multiple files to `bgm_(filename).wav` (`?f` gets replaced by the input file name):
```